spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -547,7 +547,6 @@ The object can be built via a `MetadataBuilder` with a fluent API.
 Metadata m = Metadata.builder()
     .withName("myMeter")
     .withDescription("Example meter")
-    .withType(MetricType.METER)
```

Comment:
See also lines 443-448 of this file, that discuss metric type.

Reply:
Thanks @donbourne I've pushed a further update, and also updated the changes in 5.0 section.